en/services/opendroneid.md

```diff
@@ -79,80 +107,74 @@ There is no need for this component to listen to drone ID MAVLink messages.
+> **Note** WIP: Security of drone ID data is currently quite open and it is unclear if it will be required in some locations or for some use cases.
  Most likely the transmitter components will be the ones to generate the signature for the Authentication message but the signing mechanism and how the key(s) for this is managed is open.
 
 > **Note** WIP: How will the Internet transceiver be configured? It needs to know what server(s) to connect to, credentials etc.
 
-### Open Drone ID data from other UA
-It is possible that the transmitter components also work as receivers, for obtaining drone ID data from sourrounding UAs.
-When publishing the received drone ID data as internal MAVLink messages, they must set the `compid` field to their own MAV_COMP_ID_ODID_TXRX_n ID to make it possible to distinguish this data from the drone ID data of the UA itself.
-Also the `systemid` field must be set with the system ID value that the receiver component belongs to
+### Open Drone ID data from other UA {#other_ua}
+
+It is possible that the transmitter components also work as receivers, for obtaining drone ID data from surrounding UAs.
+When publishing the received drone ID data as internal MAVLink messages, they must set the `compid` [field](https://github.com/ArduPilot/pymavlink/blob/master/generator/C/include_v2.0/mavlink_types.h#L116) to their own MAV_COMP_ID_ODID_TXRX_n ID to make it possible to distinguish this data from the drone ID data of the UA itself.
+Also the `systemid` [field](https://github.com/ArduPilot/pymavlink/blob/master/generator/C/include_v2.0/mavlink_types.h#L115) must be set with the system ID value that the receiver component belongs to.
 
-At least two possible consumers of drone ID data from sourrounding aircrafts are possible.
-- A Detect And Avoid (DAA) system will track the current and estimated future positions of the other UAs and take that into account when setting the fligt path of the UA itself.
-- A Remote ID Display (RID) application will visually show the sourrounding UA's locations (and possibly past and estimated future flight paths) to the operator of the UA, in order for him/her to utilize this information when controlling the UA.
+At least two possible consumers of drone ID data from surrounding aircrafts are possible:
+* A Detect And Avoid (DAA) system that tracks the current and estimated future positions of other UAs and takes that into account when setting the flight path of the UA itself.
+* A Remote ID Display (RID) application that visually shows the surrounding UA's locations (and possibly past and estimated future flight paths) to the operator of the UA, in order for him/her to utilize this information when controlling the UA.
 
-See below on how to combine data from other UAs.
+See [below](#combining) on how to combine remote ID data from other UAs.
 
-### Heartbeat
+### Heartbeat {#heartbeat}
```

Comment:
Just FYI, you only need to add ` {#heartbeat}` if you think the heading might change. By default markdown generators pretty much all create a default heading anchor that matches the name (including github).

Reply:
Ok. Thanks. I am still trying to learn the syntax of Markdown. I picked up on this approach from the Markdown description of the Mission Protocol microservice. 
For some of the other headings, the text is quite long and I guess it makes sense to have a shorter reference available there, in case someone wants to link to that part at some point.